docker/test/stress/run.sh

```diff
@@ -8,6 +8,9 @@ dpkg -i package_folder/clickhouse-server_*.deb
 dpkg -i package_folder/clickhouse-client_*.deb
 dpkg -i package_folder/clickhouse-test_*.deb
 
+# allow ptrace from other users
+sudo sysctl kernel.yama.ptrace_scope=0
```

Comment:
I hear about it the first time. Is it required in fuzzer as well?

Reply:
fuzzer does not need this because server does not run as a service, so the user matches and no sudo is required

Reply:
So the problem is not in yama (LSM), but the containers I guess.
So let's just not start clickhouse-server as a service and start as a regular process instead, then this should work

Reply:
Ok.